.github/workflows/scheduled-benchmarks.yml

```diff
@@ -0,0 +1,77 @@
+name: Scheduled Benchmark Tests
+on:
+  # for triggering manually, provide a token
+  workflow_dispatch:
+    inputs:
+      provider_token:
+        description: "token for remote provider"
+        required: false
+      profile_name:
+        description: "performance profile to use"
+        required: false
+      profile_filename:
+        description: "test configuration file"
+        required: false
+  # scheduled to run at every Thursday at 13:22
+  schedule:
+    - cron: '22 13 * * THU'
```

Comment:
```suggestion
    - cron: '22 13 * * *'
```

Reply:
I'm suggesting this be run daily.

Reply:
Yeah. Let's wait till we run a couple of manual triggers and have a solid process until we do it.

Reply:
Proceeding with my suggestion...